example/database.types.ts

```diff
@@ -6,7 +6,7 @@ export type Json =
   | { [key: string]: Json }
   | Json[];
 
-export interface Database {
+export type Database = {
```

Comment:
The latest Supabase lib now generates `Database` as a `type` rather than `interface`.